2024-08-30 18:00:38,795 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] WARN  [org.hib.eng.jdb.env.int.JdbcEnvironmentInitiator] (JPA Startup Thread) HHH000342: Could not obtain connection to query metadata: java.lang.NullPointerException: Cannot invoke "org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(java.sql.SQLException, String)" because the return value of "org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.sqlExceptionHelper()" is null
	at org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.doTheWork(JtaIsolationDelegate.java:186)
	at org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.lambda$delegateWork$1(JtaIsolationDelegate.java:75)
	at org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.doInSuspendedTransaction(JtaIsolationDelegate.java:107)
	at org.hibernate.resource.transaction.backend.jta.internal.JtaIsolationDelegate.delegateWork(JtaIsolationDelegate.java:72)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.getJdbcEnvironmentUsingJdbcMetadata(JdbcEnvironmentInitiator.java:276)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:107)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:68)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:130)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:263)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:238)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:52)
	at org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:136)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:247)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate.boot.internal.SessionFactoryOptionsBuilder.<init>(SessionFactoryOptionsBuilder.java:276)
	at io.quarkus.hibernate.orm.runtime.recording.PrevalidatedQuarkusMetadata.buildSessionFactoryOptionsBuilder(PrevalidatedQuarkusMetadata.java:70)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:84)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:72)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:157)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:64)
	at java.base/java.lang.Thread.run(Thread.java:840)

2024-08-30 18:00:33,997 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) entering args (oracle.jdbc.internal.AbstractConnectionBuilder$1@236a36e1)
2024-08-30 18:00:33,998 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) traceId=225F7909. 
2024-08-30 18:00:37,999 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) throwing: java.sql.SQLRecoverableException: ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1441)
	at oracle.jdbc.driver.T4CConnection.handleLogonInterruptedIOException(T4CConnection.java:1309)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:986)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.io.IOException: , Authentication lapse 0 ms.
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1436)
	... 14 more
Caused by: java.io.InterruptedIOException: 
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:299)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more
Caused by: java.io.InterruptedIOException: Socket read interrupted
	at oracle.net.nt.TimeoutSocketChannel.doSelect(TimeoutSocketChannel.java:706)
	at oracle.net.nt.TimeoutSocketChannel.initializeSocketChannel(TimeoutSocketChannel.java:245)
	at oracle.net.nt.TimeoutSocketChannel.connect(TimeoutSocketChannel.java:203)
	at oracle.net.nt.TimeoutSocketChannel.<init>(TimeoutSocketChannel.java:170)
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:256)
	... 21 more

2024-08-30 18:00:38,000 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) properties={LOCALE=en_US, DriverVersion=23.3.0.23.09, java.library.path: =/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib, java.class.path: =/home/pmorales/Downloads/ProyectoARQ/ARQ/Hotel/backend/sistema-hoteles-be/target/sistema-hoteles-be-dev.jar, java.version: =17.0.12}. 
2024-08-30 18:00:38,822 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] WARN  [io.agr.pool] (agroal-11) Datasource '<default>': ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
2024-08-30 18:00:44,582 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] ERROR [org.hib.que.sqm.mut.int.tem.GlobalTemporaryTableStrategy] (JPA Startup Thread) Unable obtain JDBC Connection: java.sql.SQLException: Acquisition timeout while waiting for new connection
	at io.agroal.pool.ConnectionPool.handlerFromSharedCache(ConnectionPool.java:321)
	at io.agroal.pool.ConnectionPool.getConnection(ConnectionPool.java:249)
	at io.agroal.pool.DataSource.getConnection(DataSource.java:86)
	at io.quarkus.hibernate.orm.runtime.customized.QuarkusConnectionProvider.getConnection(QuarkusConnectionProvider.java:23)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess.obtainConnection(JdbcEnvironmentInitiator.java:428)
	at org.hibernate.query.sqm.mutation.internal.temptable.GlobalTemporaryTableStrategy.prepare(GlobalTemporaryTableStrategy.java:90)
	at org.hibernate.persister.entity.AbstractEntityPersister.lambda$prepareMappingModel$14(AbstractEntityPersister.java:5084)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess$PostInitCallbackEntry.process(MappingModelCreationProcess.java:246)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.executePostInitCallbacks(MappingModelCreationProcess.java:106)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.execute(MappingModelCreationProcess.java:88)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.process(MappingModelCreationProcess.java:42)
	at org.hibernate.metamodel.model.domain.internal.MappingMetamodelImpl.finishInitialization(MappingMetamodelImpl.java:198)
	at org.hibernate.internal.SessionFactoryImpl.initializeMappingModel(SessionFactoryImpl.java:364)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:293)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:87)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:72)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:157)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:64)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at io.agroal.pool.ConnectionPool.handlerFromSharedCache(ConnectionPool.java:298)
	... 20 more

2024-08-30 18:00:38,999 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) entering args (oracle.jdbc.internal.AbstractConnectionBuilder$1@3b10f061)
2024-08-30 18:00:39,000 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) traceId=77E32377. 
2024-08-30 18:00:43,999 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) throwing: java.sql.SQLRecoverableException: ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1441)
	at oracle.jdbc.driver.T4CConnection.handleLogonInterruptedIOException(T4CConnection.java:1309)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:986)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.io.IOException: , Authentication lapse 0 ms.
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1436)
	... 14 more
Caused by: java.io.InterruptedIOException: 
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:299)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more
Caused by: java.io.InterruptedIOException: Socket read interrupted
	at oracle.net.nt.TimeoutSocketChannel.doSelect(TimeoutSocketChannel.java:706)
	at oracle.net.nt.TimeoutSocketChannel.initializeSocketChannel(TimeoutSocketChannel.java:245)
	at oracle.net.nt.TimeoutSocketChannel.connect(TimeoutSocketChannel.java:203)
	at oracle.net.nt.TimeoutSocketChannel.<init>(TimeoutSocketChannel.java:170)
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:256)
	... 21 more

2024-08-30 18:00:44,000 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) properties={LOCALE=en_US, DriverVersion=23.3.0.23.09, java.library.path: =/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib, java.class.path: =/home/pmorales/Downloads/ProyectoARQ/ARQ/Hotel/backend/sistema-hoteles-be/target/sistema-hoteles-be-dev.jar, java.version: =17.0.12}. 
2024-08-30 18:00:44,600 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] WARN  [io.agr.pool] (agroal-11) Datasource '<default>': ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
2024-08-30 18:00:44,001 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) entering args (oracle.jdbc.internal.AbstractConnectionBuilder$1@58789e09)
2024-08-30 18:00:49,590 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] ERROR [org.hib.que.sqm.mut.int.tem.GlobalTemporaryTableStrategy] (JPA Startup Thread) Unable obtain JDBC Connection: java.sql.SQLException: Acquisition timeout while waiting for new connection
	at io.agroal.pool.ConnectionPool.handlerFromSharedCache(ConnectionPool.java:321)
	at io.agroal.pool.ConnectionPool.getConnection(ConnectionPool.java:249)
	at io.agroal.pool.DataSource.getConnection(DataSource.java:86)
	at io.quarkus.hibernate.orm.runtime.customized.QuarkusConnectionProvider.getConnection(QuarkusConnectionProvider.java:23)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess.obtainConnection(JdbcEnvironmentInitiator.java:428)
	at org.hibernate.query.sqm.mutation.internal.temptable.GlobalTemporaryTableStrategy.prepare(GlobalTemporaryTableStrategy.java:90)
	at org.hibernate.persister.entity.AbstractEntityPersister.lambda$prepareMappingModel$14(AbstractEntityPersister.java:5084)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess$PostInitCallbackEntry.process(MappingModelCreationProcess.java:246)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.executePostInitCallbacks(MappingModelCreationProcess.java:106)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.execute(MappingModelCreationProcess.java:88)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.process(MappingModelCreationProcess.java:42)
	at org.hibernate.metamodel.model.domain.internal.MappingMetamodelImpl.finishInitialization(MappingMetamodelImpl.java:198)
	at org.hibernate.internal.SessionFactoryImpl.initializeMappingModel(SessionFactoryImpl.java:364)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:293)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:87)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:72)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:157)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:64)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at io.agroal.pool.ConnectionPool.handlerFromSharedCache(ConnectionPool.java:298)
	... 20 more

2024-08-30 18:00:44,002 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) traceId=61AA4DF7. 
2024-08-30 18:00:49,000 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) throwing: java.sql.SQLRecoverableException: ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1441)
	at oracle.jdbc.driver.T4CConnection.handleLogonInterruptedIOException(T4CConnection.java:1309)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:986)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.io.IOException: , Authentication lapse 0 ms.
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1436)
	... 14 more
Caused by: java.io.InterruptedIOException: 
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:299)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more
Caused by: java.io.InterruptedIOException: Socket read interrupted
	at oracle.net.nt.TimeoutSocketChannel.doSelect(TimeoutSocketChannel.java:706)
	at oracle.net.nt.TimeoutSocketChannel.initializeSocketChannel(TimeoutSocketChannel.java:245)
	at oracle.net.nt.TimeoutSocketChannel.connect(TimeoutSocketChannel.java:203)
	at oracle.net.nt.TimeoutSocketChannel.<init>(TimeoutSocketChannel.java:170)
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:256)
	... 21 more

2024-08-30 18:00:49,001 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) properties={LOCALE=en_US, DriverVersion=23.3.0.23.09, java.library.path: =/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib, java.class.path: =/home/pmorales/Downloads/ProyectoARQ/ARQ/Hotel/backend/sistema-hoteles-be/target/sistema-hoteles-be-dev.jar, java.version: =17.0.12}. 
2024-08-30 18:00:49,600 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] WARN  [io.agr.pool] (agroal-11) Datasource '<default>': ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
2024-08-30 18:00:54,597 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] ERROR [org.hib.que.sqm.mut.int.tem.GlobalTemporaryTableStrategy] (JPA Startup Thread) Unable obtain JDBC Connection: java.sql.SQLException: Acquisition timeout while waiting for new connection
	at io.agroal.pool.ConnectionPool.handlerFromSharedCache(ConnectionPool.java:321)
	at io.agroal.pool.ConnectionPool.getConnection(ConnectionPool.java:249)
	at io.agroal.pool.DataSource.getConnection(DataSource.java:86)
	at io.quarkus.hibernate.orm.runtime.customized.QuarkusConnectionProvider.getConnection(QuarkusConnectionProvider.java:23)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess.obtainConnection(JdbcEnvironmentInitiator.java:428)
	at org.hibernate.query.sqm.mutation.internal.temptable.GlobalTemporaryTableStrategy.prepare(GlobalTemporaryTableStrategy.java:90)
	at org.hibernate.persister.entity.AbstractEntityPersister.lambda$prepareMappingModel$14(AbstractEntityPersister.java:5084)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess$PostInitCallbackEntry.process(MappingModelCreationProcess.java:246)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.executePostInitCallbacks(MappingModelCreationProcess.java:106)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.execute(MappingModelCreationProcess.java:88)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.process(MappingModelCreationProcess.java:42)
	at org.hibernate.metamodel.model.domain.internal.MappingMetamodelImpl.finishInitialization(MappingMetamodelImpl.java:198)
	at org.hibernate.internal.SessionFactoryImpl.initializeMappingModel(SessionFactoryImpl.java:364)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:293)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:87)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:72)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:157)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:64)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at io.agroal.pool.ConnectionPool.handlerFromSharedCache(ConnectionPool.java:298)
	... 20 more

2024-08-30 18:00:49,002 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) entering args (oracle.jdbc.internal.AbstractConnectionBuilder$1@7763e31f)
2024-08-30 18:00:49,003 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) traceId=3A2BAB92. 
2024-08-30 18:00:53,999 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) throwing: java.sql.SQLRecoverableException: ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1441)
	at oracle.jdbc.driver.T4CConnection.handleLogonInterruptedIOException(T4CConnection.java:1309)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:986)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.io.IOException: , Authentication lapse 0 ms.
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1436)
	... 14 more
Caused by: java.io.InterruptedIOException: 
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:299)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more
Caused by: java.io.InterruptedIOException: Socket read interrupted
	at oracle.net.nt.TimeoutSocketChannel.doSelect(TimeoutSocketChannel.java:706)
	at oracle.net.nt.TimeoutSocketChannel.initializeSocketChannel(TimeoutSocketChannel.java:245)
	at oracle.net.nt.TimeoutSocketChannel.connect(TimeoutSocketChannel.java:203)
	at oracle.net.nt.TimeoutSocketChannel.<init>(TimeoutSocketChannel.java:170)
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:256)
	... 21 more

2024-08-30 18:00:54,000 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) properties={LOCALE=en_US, DriverVersion=23.3.0.23.09, java.library.path: =/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib, java.class.path: =/home/pmorales/Downloads/ProyectoARQ/ARQ/Hotel/backend/sistema-hoteles-be/target/sistema-hoteles-be-dev.jar, java.version: =17.0.12}. 
2024-08-30 18:00:54,606 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] WARN  [io.agr.pool] (agroal-11) Datasource '<default>': ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
2024-08-30 18:00:59,602 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] ERROR [org.hib.que.sqm.mut.int.tem.GlobalTemporaryTableStrategy] (JPA Startup Thread) Unable obtain JDBC Connection: java.sql.SQLException: Acquisition timeout while waiting for new connection
	at io.agroal.pool.ConnectionPool.handlerFromSharedCache(ConnectionPool.java:321)
	at io.agroal.pool.ConnectionPool.getConnection(ConnectionPool.java:249)
	at io.agroal.pool.DataSource.getConnection(DataSource.java:86)
	at io.quarkus.hibernate.orm.runtime.customized.QuarkusConnectionProvider.getConnection(QuarkusConnectionProvider.java:23)
	at org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator$ConnectionProviderJdbcConnectionAccess.obtainConnection(JdbcEnvironmentInitiator.java:428)
	at org.hibernate.query.sqm.mutation.internal.temptable.GlobalTemporaryTableStrategy.prepare(GlobalTemporaryTableStrategy.java:90)
	at org.hibernate.persister.entity.AbstractEntityPersister.lambda$prepareMappingModel$14(AbstractEntityPersister.java:5084)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess$PostInitCallbackEntry.process(MappingModelCreationProcess.java:246)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.executePostInitCallbacks(MappingModelCreationProcess.java:106)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.execute(MappingModelCreationProcess.java:88)
	at org.hibernate.metamodel.mapping.internal.MappingModelCreationProcess.process(MappingModelCreationProcess.java:42)
	at org.hibernate.metamodel.model.domain.internal.MappingMetamodelImpl.finishInitialization(MappingMetamodelImpl.java:198)
	at org.hibernate.internal.SessionFactoryImpl.initializeMappingModel(SessionFactoryImpl.java:364)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:293)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:87)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:72)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:157)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:64)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at io.agroal.pool.ConnectionPool.handlerFromSharedCache(ConnectionPool.java:298)
	... 20 more

2024-08-30 18:00:54,001 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) entering args (oracle.jdbc.internal.AbstractConnectionBuilder$1@113c3dda)
2024-08-30 18:00:54,002 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) traceId=7A2881ED. 
2024-08-30 18:00:58,999 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) throwing: java.sql.SQLRecoverableException: ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1441)
	at oracle.jdbc.driver.T4CConnection.handleLogonInterruptedIOException(T4CConnection.java:1309)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:986)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.io.IOException: , Authentication lapse 0 ms.
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1436)
	... 14 more
Caused by: java.io.InterruptedIOException: 
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:299)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more
Caused by: java.io.InterruptedIOException: Socket read interrupted
	at oracle.net.nt.TimeoutSocketChannel.doSelect(TimeoutSocketChannel.java:706)
	at oracle.net.nt.TimeoutSocketChannel.initializeSocketChannel(TimeoutSocketChannel.java:245)
	at oracle.net.nt.TimeoutSocketChannel.connect(TimeoutSocketChannel.java:203)
	at oracle.net.nt.TimeoutSocketChannel.<init>(TimeoutSocketChannel.java:170)
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:256)
	... 21 more

2024-08-30 18:00:59,000 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) properties={LOCALE=en_US, DriverVersion=23.3.0.23.09, java.library.path: =/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib, java.class.path: =/home/pmorales/Downloads/ProyectoARQ/ARQ/Hotel/backend/sistema-hoteles-be/target/sistema-hoteles-be-dev.jar, java.version: =17.0.12}. 
2024-08-30 18:00:59,609 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] WARN  [io.agr.pool] (agroal-11) Datasource '<default>': ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
2024-08-30 18:01:05,007 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (JPA Startup Thread) SQL Error: 0, SQLState: null
2024-08-30 18:00:59,998 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) entering args (oracle.jdbc.internal.AbstractConnectionBuilder$1@48ed7e40)
2024-08-30 18:00:59,999 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) traceId=394ECF17. 
2024-08-30 18:01:05,008 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] ERROR [org.hib.eng.jdb.spi.SqlExceptionHelper] (JPA Startup Thread) Acquisition timeout while waiting for new connection
2024-08-30 18:01:04,999 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) throwing: java.sql.SQLRecoverableException: ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1441)
	at oracle.jdbc.driver.T4CConnection.handleLogonInterruptedIOException(T4CConnection.java:1309)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:986)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1157)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:825)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:651)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:226)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.io.IOException: , Authentication lapse 0 ms.
	at oracle.jdbc.driver.T4CConnection.handleLogonIOException(T4CConnection.java:1436)
	... 14 more
Caused by: java.io.InterruptedIOException: 
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:299)
	at oracle.net.nt.TcpNTAdapter.connect(TcpNTAdapter.java:222)
	at oracle.net.nt.ConnOption.connect(ConnOption.java:327)
	at oracle.net.nt.ConnStrategy.executeConnOption(ConnStrategy.java:1144)
	at oracle.net.nt.ConnStrategy.execute(ConnStrategy.java:718)
	at oracle.net.resolver.AddrResolution.resolveAndExecute(AddrResolution.java:707)
	at oracle.net.ns.NSProtocol.establishConnection(NSProtocol.java:938)
	at oracle.net.ns.NSProtocol.connect(NSProtocol.java:327)
	at oracle.jdbc.driver.T4CConnection.connectNetworkSessionProtocol(T4CConnection.java:3185)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:926)
	... 12 more
Caused by: java.io.InterruptedIOException: Socket read interrupted
	at oracle.net.nt.TimeoutSocketChannel.doSelect(TimeoutSocketChannel.java:706)
	at oracle.net.nt.TimeoutSocketChannel.initializeSocketChannel(TimeoutSocketChannel.java:245)
	at oracle.net.nt.TimeoutSocketChannel.connect(TimeoutSocketChannel.java:203)
	at oracle.net.nt.TimeoutSocketChannel.<init>(TimeoutSocketChannel.java:170)
	at oracle.net.nt.TcpNTAdapter.establishSocket(TcpNTAdapter.java:256)
	... 21 more

2024-08-30 18:01:05,000 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] INFO  [ora.jdbc] (agroal-11) properties={LOCALE=en_US, DriverVersion=23.3.0.23.09, java.library.path: =/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib, java.class.path: =/home/pmorales/Downloads/ProyectoARQ/ARQ/Hotel/backend/sistema-hoteles-be/target/sistema-hoteles-be-dev.jar, java.version: =17.0.12}. 
2024-08-30 18:01:05,013 pmorales-virtual-machine sistema-hoteles-be-dev.jar[7943] WARN  [io.agr.pool] (agroal-11) Datasource '<default>': ORA-17002: I/O error: , Authentication lapse 0 ms.
https://docs.oracle.com/error-help/db/ora-17002/
